apps/site/lib/site_web/templates/fare_transformation/proposed_sales_locations.html.eex

```diff
@@ -28,6 +28,11 @@
       <%= form_for @conn, @conn.request_path<>"#address-search-input", [as: :location, method: :get, class: "form"], fn _f -> %>
         <%= input_location(%InputLocation{id: "address-search-input", name: :location, name_index: :address, address: @address, address_error: get_flash(@conn, :info), submit: true, required: true, placeholder: "Enter a location", button_class: ""}) %>
       <% end %>
+      <%= case @nearby_proposed_locations do %>
```

Comment:
If there is only one `case`, you could probably change it to an `if`.

Reply:
I also have to match on empty array vs anything else, and I couldn't figure out a more elegant way to do that 😅

Reply:
Do you mean the "anything else" in line 34? the `end` in line 35 makes it a 'do nothing'. Let me know if I'm not following!

Reply:
Lines 31-35 should only be rendered when `@nearby_proposed_locations` exists as an empty array.  I can't seem to include a matching statement in an .eex if-statement.  For example, `<%= if @nearby_proposed_locations = [] do %>` is not valid.  But I can use "case" to do the match.

Lines 48-58 do a similar thing.  I need to use "case" to weed out @nearby_proposed_locations = [], and then in all other cases the section is rendered.

Does that make sense?

Reply:
Can you use `if Enum.empty?(@nearby_proposed_locations)` maybe?

Reply:
Ohoo yep!!  That works!